rhscl-mariadb-apb/apb.yml

```diff
@@ -0,0 +1,47 @@
+id: a8e652c1-11e0-4202-b6df-09517f75ef9a
+name: rhscl-mariadb-apb
+image: registry.access.redhat.com/rhscl/mariadb-101-rhel7
+#image: ansibleplaybookbundle/rhscl-mariadb-apb
+description: mariadb apb implementation
+bindable: true
+async: optional
+tags:
+  - database
+  - rhscl-mariadb
```

Comment:
Not certain on this one, but if you want it correctly sorted in the UI I'd guess it needs to be mariadb.